docs-src/cloud/what-integrations-are-supported-by-export.md

```diff
@@ -0,0 +1,14 @@
+---
+id: what-integrations-are-supported-by-export
+title: Supported integrations for Workflow History Export
+sidebar_label: Supported integrations
+description: Workflow History Export supports Amazon Simple Storage Service (S3).
+tags:
+  - temporal cloud
+  - operations
+---
+
+The Workflow History Export feature allows users to export Closed Workflow Histories to [Amazon Simple Storage Service (S3)](https://docs.aws.amazon.com/s3/) as the destination for exported Workflow Histories.
```

Comment:
The end of this sentence feels a bit redundant. "The Workflow History Export feature allows users to export Closed Workflow Histories to [Amazon Simple Storage Service (S3)](https://docs.aws.amazon.com/s3/) as the destination." This should be enough imo